### DISCUSSION TITLE
what errors do you see in network tab of [Jan 17]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4924] [Discord Threads]
> 💁 **o1lab**

**Comment by o1lab** [comment]
> 🙋‍♂️ **VetaBelina**  
####   
![image0](https://cdn.discordapp.com/attachments/1049415336438546432/1050070275187806269/image.png)  
#  
> 🙋‍♂️ **VetaBelina**  
####   
![image0](https://cdn.discordapp.com/attachments/1049415336438546432/1050072005480824862/image.png)  
#  
> 🙋‍♂️ **VetaBelina**  
#### <@310157751395287040> Hello! Could you, please, tell me what to do with that?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1049415336438546432)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
